src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,18 @@
 package mate.academy
 
+import mate.academy.service.FigureSupplier
+
+const val FIGURES_NUMBER_FOR_RENDER = 10
+
 fun main() {
-    // draw all figures here
+    val figureSupplier = FigureSupplier()
+    val middleIndex = FIGURES_NUMBER_FOR_RENDER / 2
+    for (i in 0..FIGURES_NUMBER_FOR_RENDER) {
```

Comment:
The loop condition `0..FIGURES_NUMBER_FOR_RENDER` will iterate `FIGURES_NUMBER_FOR_RENDER + 1` times, which is one more than intended. Use `until` instead of `..` to iterate the correct number of times: `for (i in 0 until FIGURES_NUMBER_FOR_RENDER)`.